lib/spoom/sorbet/errors.rb

```diff
@@ -118,21 +122,32 @@ class Error
         sig { returns(T::Array[String]) }
         attr_reader :more
 
+        # Other files associated with the error
+        sig { returns(T::Array[String]) }
+        attr_reader :related_files
+
         sig do
           params(
             file: T.nilable(String),
             line: T.nilable(Integer),
             message: T.nilable(String),
             code: T.nilable(Integer),
-            more: T::Array[String]
+            more: T::Array[String],
           ).void
         end
-        def initialize(file, line, message, code, more = [])
+        def initialize(
+          file,
+          line,
+          message,
+          code,
+          more = []
+        )
           @file = file
           @line = line
           @message = message
           @code = code
           @more = more
+          @related_files = T.let([], T::Array[String])
```

Comment:
Should we make this a set? We don't need to check the same path twice if it appears in multiple error sections.

Reply:
Maybe we should also ignore `related_files` that are the error `file` itself like in autocorrect sections or some errors like this one: https://github.com/Shopify/spoom/blob/main/test/spoom/sorbet/errors_test.rb#L125

Reply:
I made this change, let me know what ya think.